lib/Models/WebMapServiceCatalogItem.ts

```diff
@@ -1187,6 +1184,10 @@ class WebMapServiceCatalogItem
         parameters: parameters,
         getFeatureInfoParameters: {
           ...dimensionParameters,
+          feature_count:
+            1 +
+            (this.maximumShownFeatureInfos ||
+              this.terria.configParameters.defaultMaximumShownFeatureInfos)!,
```

Comment:
Thanks so much for fixing this @zoran995 !

Can you please use nullish coalescing instead of logical OR here?

```suggestion
            1 +
            (this.maximumShownFeatureInfos ??
              this.terria.configParameters.defaultMaximumShownFeatureInfos)
```

And can you please change the type of `this.terria.configParameters.defaultMaximumShownFeatureInfos` to be `number` instead of `number | undefined`

https://github.com/zoran995/terriajs-i18n/blob/a878b51da78cb8756cf2f78ed20552a3d4197c48/lib/Models/Terria.ts#L103-L106

Reply:
changing the type of `defaultMaximumShownFeatureInfos` to be number will require an update of tests depending on Terria, if I recall correctly? Maybe handle this in separate PR to handle all `configParameters` that have a default value (half of them)

Reply:
actually, this would be better? maximumShownFeatureInfos is coming from user input and it can be anything
```
(parseInt(this.maximumShownFeatureInfos) ||
              this.terria.configParameters.defaultMaximumShownFeatureInfos)
```

Reply:
The type of trait values is validated when it is updated/upserted from JSON

https://github.com/TerriaJS/terriajs/blob/33f03c0477e56ba7d7a4cc40771406cf5fc801e2/lib/Traits/primitiveTrait.ts#L44-L62

So it should only be `number | undefined`

Reply:
Also just checked, you should be able to change this.terria.configParameters.defaultMaximumShownFeatureInfos to type `number` without breaking tests.

Reply:
done